src/patternfly/components/CustomSearchInput/examples/CustomSearchInput.md

```diff
@@ -0,0 +1,48 @@
+---
+title: 'Custom search input'
+beta: true
+section: components
+cssPrefix: pf-c-custom-search-input
+---
+
+## Examples
+```hbs title=Basic
+{{#> custom-search-input custom-search-input--placeholder="Find by name"}}
+{{/custom-search-input}}
+```
+
+```hbs title=No-match
+{{#> custom-search-input custom-search-input--placeholder="Find by name" custom-search-input--value="Joh"}}
+{{/custom-search-input}}
+```
+
+```hbs title=Match-with-result-count
+{{#> custom-search-input custom-search-input--placeholder="Find by name" custom-search-input--value="John Doe" custom-search-input--count="3"}}
+{{/custom-search-input}}
+```
+
+```hbs title=Match-with-navigable-options
```

Comment:
Can you add spaces between the 1 and / and the / and 3 so its like: 

<img width="118" alt="Screen Shot 2020-07-09 at 10 27 08 AM" src="https://user-images.githubusercontent.com/20118816/87052477-d50c5380-c1ce-11ea-9f6e-ff8443bd3e33.png">